recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -130,7 +125,12 @@ def _configure_meson(self):
         defs["builtin_loaders"] = "all"
         defs["gio_sniffing"] = "false"
         defs["introspection"] = "enabled" if self.options.with_introspection else "disabled"
-        args=[]
+        args = []
+        # Workaround for https://bugs.llvm.org/show_bug.cgi?id=16404
+        # Ony really for the purporses of building on CCI - end users can
+        # workaround this by appropriately setting global linker flags in their profile
+        if self.settings.compiler == "clang":
```

Comment:
link against  "compiler-rt" only if clang and debug mode, since some distributions don't ship "compiler-rt" with clang, and this might cause it to unnecessarily fail in release mode.